src/shared/sources/tw/index.js

```diff
@@ -0,0 +1,48 @@
+const assert = require('assert')
+const maintainers = require('../_lib/maintainers.js')
+const parse = require('../_lib/parse.js')
+
+const country = `iso1:TW`
+
+const schemaKeysByHeading = {
+  送驗: 'tested',
+  排除: null, // Negative Tests
+  確診: 'cases',
+  死亡: 'deaths',
+  解除隔離: 'recovered',
+}
+
+module.exports = {
+  country,
+  priority: 1,
+  friendly: {
+    name: 'Taiwan CDC',
+    url: 'https://cdc.gov.tw/'
+  },
+  maintainers: [ maintainers.camjc ],
+  scrapers: [
+    {
+      startDate: '2020-01-15',
```

Comment:
Actually I dunno when this goes back to. Should we set today/nothing/pick something?

Reply:
Today: if it is timeseries we will never get data before the date you set. If not timeseries, date of merge is fine.